src/relax_ng_validator.cc

```diff
@@ -81,9 +81,9 @@ bool RelaxNGValidator::Validate(const xmlpp::Document* doc) {
     throw ValidationError(e.message);
   }
 
-  if (res != 0) {
-    throw ValidationError("Document failed schema validation");
-  }
+  // if (res != 0) {
+  //  throw ValidationError("Document failed schema validation");
+  // }
```

Comment:
Are these commented lines meant to be included in this PR?

Reply:
No, it was not. And it's probably why the tests are failing.